src/test/java/testresults/PutTestResultsCertificateGeneration.java

```diff
@@ -19,22 +20,23 @@
 import util.JsonPathAlteration;
 import org.apache.http.HttpStatus;
 
-import java.text.SimpleDateFormat;
+import java.text.SimpleDateFormat;;
```

Comment:
can the extra semi colon be removed please

Reply:
Done